messages/smarc_control_msgs/msg/Topics.msg

```diff
@@ -9,6 +9,8 @@
 ########################
 # States
 string STATES = 'core/odom_gt' # Type: Odometry
+string DEPTH = 'dr/depth' # Type: Float64
+string PITCH = 'core/imu' # Type: 
```

Comment:
a bit suspect :)

Reply:
Just FYI, some of these topics (like odom and depth) WILL move to a common(to all smarc robots) topics list _soon_. I am basically leaving SAM stuff alone until "it works". There WILL be a day of clean-up ;)